visualizer.js

```diff
@@ -218,6 +214,9 @@ function onSample (sample) {
             offsetForGrid(sample.channelData[index], index)
         );
         channel.shift();
+        channel = notchFilter.multiStep(channel);
+        channel = lpFilter.multiStep(channel);
+        channel = hpFilter.multiStep(channel);
```

Comment:
I think it's doing it filter every window refresh since it is governed by `if (sampleNumber === bins)` but I do agree that we should reformulate the time series code. also, it would be nice to move all the time-frequency stuff to `fili.js`.